### PR TITLE
fix: copy link button

### LINKED ISSUE #15: Copy link button works only once in Firefox
The Javascript functions to copy markdown links to the clipboard only work the first time you click on it. After, the toast is displayed but the correct text is not copied to the clipboard unless you reload the page.

### PR DESCRIPTION
**Pull request by PascalRepond**
- Closes #15

Co-Authored-by: Pascal Repond <pascal.repond@rero.ch>

### CODE REVIEW

File: flask_wiki/static/js/wiki.js [hunk 1]
```diff
@@ -49,39 +49,17 @@ $(document).ready(function () {
...
-      .select()
-      .on('focus', function () {
-        document.execCommand('selectAll', false, null);
-      })
-      .focus();
-    document.execCommand('copy');
-    $temp.remove();
-    $('.toast').toast('show');
+    var text = '![' + name + '](' + link + ' "' + name + '"' + ')';
+    copyToClipboard(text);
```
Comment (Garfield-fr) [inline flask_wiki/static/js/wiki.js]: Check with this syntax: ```copyToClipboard(`${name}`) ```

File: flask_wiki/static/js/wiki.js [hunk 2]
```diff
@@ -49,39 +49,17 @@ $(document).ready(function () {
...
-      .select()
-      .on('focus', function () {
-        document.execCommand('selectAll', false, null);
-      })
-      .focus();
-    document.execCommand('copy');
-    $temp.remove();
-    $('.toast').toast('show');
+    var text = '[' + name + '](' + link + ')';
+    copyToClipboard(text);
```
Comment (Garfield-fr) [inline flask_wiki/static/js/wiki.js]: Check with this syntax: ``` copyToClipboard(`[${name}](${link})`) ```

File: flask_wiki/static/js/wiki.js [hunk 3]
```diff
@@ -93,6 +71,17 @@ $(document).ready(function () {
   });
 });
 
+function copyToClipboard(text) {
+  navigator.clipboard.writeText(text).then(
+    () => {
+      $('.toast').toast('show');
+    },
+    () => {
+      console.log('Problem when copying the link to the clipboard.');
```
Comment (Garfield-fr) [inline flask_wiki/static/js/wiki.js]: I'm not a fan of putting a console here.
A silent error?

**Review by Garfield-fr — APPROVED** [review]
Why do you have translations in this PR?

File: flask_wiki/static/js/wiki.js [hunk 4]
```diff
@@ -48,40 +48,17 @@ $(document).ready(function () {
...
-      .html('[' + name + '](' + link + ')')
-      .select()
-      .on('focus', function () {
-        document.execCommand('selectAll', false, null);
-      })
-      .focus();
-    document.execCommand('copy');
-    $temp.remove();
-    $('.toast').toast('show');
+    var text = '[' + name + '](' + link + ')';
```
Comment (Garfield-fr) [inline flask_wiki/static/js/wiki.js]: Remove this line ?

**Review by Garfield-fr — APPROVED** [review]

**Review by jma — APPROVED** [review]
Great!

**Pull request merged**